admin/app/controllers/authenticated/certifications/sessions/info/list.js

```diff
@@ -1,30 +1,34 @@
-import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
+import Controller from '@ember/controller';
 import ENV from 'pix-admin/config/environment';
-import EmberObject from '@ember/object';
+import EmberObject, { action } from '@ember/object';
 import _ from 'lodash';
 
-export default Controller.extend({
+export default class ListController extends Controller {
+  @service
+  session;
+
+  @service
+  sessionInfoService;
 
-  session: service(),
-  sessionInfoService: service(),
-  notifications: service('notification-messages'),
+  @service
+  notifications;
 
-  displayConfirm: false,
-  displaySessionReport: false,
-  confirmMessage: null,
-  confirmAction: null,
-  showSelectedActions: false,
-  selectedCertifications: null,
-  certificationsInSessionReport: null,
+  displayConfirm = false;
+  displaySessionReport = false;
+  confirmMessage = null;
+  confirmAction = null;
+  showSelectedActions = false;
+  selectedCertifications = null;
+  certificationsInSessionReport = null;
 
   init() {
-    this._super();
+    super.init();
     this.selected = [];
     this.set('certificationsInSessionReport', []);
```

Comment:
Utiliser le constructeur plutôt que le init.

Reply:
- this.selected supprimé (car obsolète).
- init() supprimé